app/models/item.rb

```diff
@@ -27,4 +27,9 @@ def status
       "Inactive"
     end
   end
+
+  scope :active,
```

Comment:
Maybe put these scopes on the same line? It doesn't look they would go over 80 characters.